src/js/routes.js

```diff
@@ -579,6 +579,15 @@ angular.module('copayApp').config(function(historicLogProvider, $provide, $logPr
           }
         }
       })
+      .state('tabs.preferences.preferencesExternal', {
```

Comment:
I don't see the point here for this view. I connected/disconnected both devices and always have "No hardware information available". Is this necessary for trezor or ledger?